extensions/amp-a4a/0.1/amp-ad-utils.js

```diff
@@ -162,6 +163,24 @@ export function getAmpAdMetadata(creative) {
   }
 }
 
+/**
+ * Merges any elements from customElementExtensions array into extensions array if
+ * the element is not present.
+ * @param {!Array<{custom-element: string, 'src': string}} extensions
+ * @param {!Array<string>} customElementExtensions
+ */
+export function mergeExtensionsMetadata(extensions, customElementExtensions) {
+  for (let i = 0; i < customElementExtensions.length; i++) {
+    const extensionId = customElementExtensions[i];
+    if (!extensionsHasElement(extensions, extensionId)) {
+      extensions.push({
+        'custom-element': extensionId,
+        src: `${urls.cdn}/v0/${extensionId}-latest.js`,
```

Comment:
we talked about possibly making this `0.1` to keep current behavior, but I felt `latest` seems like a better future solution. Happy to change if people feel otherwise.

Reply:
I think we have to go with `0.1` at this time for backward compatibility. The issue here is that previously providing just `amp-ext` would always load `0.1` and that's the component that was used for testing and all. Changing to `latest` is too dangerous. The `amp-mustache` is a slightly special case. But I'd even use `0.1` for it to be consistent.

That means that if a creative/extension/server/etc wants to be more specific, they simply need to switch to a more precise `extensions` field.

Reply:
changed to `0.1`.